map/generate-offline-map-with-local-basemap/settings.gradle

```diff
@@ -1 +1,12 @@
 rootProject.name = 'generate-offline-map-with-local-basemap'
+
+// download sample data
+def ant = new groovy.util.AntBuilder()
+
+if (!file("./samples-data").exists()) {
+    file("./samples-data/naperville").mkdirs()
+    ant.get(
+            src: "https://arcgisruntime.maps.arcgis.com/sharing/rest/content/items/628e8e3521cf45e9a28a12fe10c02c4d/data",
+            dest: file("./samples-data/naperville")
```

Comment:
I think it needs a specification of the true file name here, otherwise it downloads a `data` file without extension.
`dest: file("./samples-data/naperville/naperville_imagery.tpk")`
I will try and find the rest of the samples that work this way and point them out. I think it's only an issue when the files are not archived.